packages/utils/src/function/__tests__/function.test.ts

```diff
@@ -1,3 +1,5 @@
+jest.useFakeTimers();
```

Comment:
fake timers should be part of global jest configuration in package.json

Reply:
moved